content/docs/getting-started/backends.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Working with Backends
+prev: /docs/getting-started/edit-content
+next: null
+---
+
+You may have noticed that content changes don't persist on refresh. Every time the page loads, the `data` populates with the form's initial values. In order for the CMS to be useful, it needs to track and persist data changes with some sort of backend.
+
+## Loading Content from an external API
+
+For this example, we will use a [fake API](https://jsonplaceholder.typicode.com/) to fetch content and post changes. We will use the `loadInitialValues` function to fetch content.
+
+**src/App.js**
+
+```diff
+//..
+
```

Comment:
```suggestion
- const pageData = {
-   title: 'Tina is not a CMS',
-   body: 'It is a toolkit for creating a custom CMS.',
- }
```

(hopefully that will render correctly)

We could consider removing this chunk of code in the diff in case people have an eslint rule that causes a compile error for unused code